cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -25,20 +25,33 @@
 # CMP - 9531
 
 
-@pytest.mark.parametrize(('prop', 'rel'), product(REPLICATORS_PROPERTIES_FIELDS,
-                                                  REPLICATORS_RELATIONSHIPS_FIELDS))
-def test_replicators_properties(provider, prop, rel):
+@pytest.mark.parametrize('prop', product(REPLICATORS_PROPERTIES_FIELDS))
```

Comment:
```product(X)``` doesn't necessarily == ```X```
as can be seen below
```
In [11]: product(REPLICATORS_PROPERTIES_FIELDS) == REPLICATORS_PROPERTIES_FIELDS
Out[11]: False
```
However, I know what you are trying to say, that essentially they are the same and yes in this case, we can remove ```product(REPLICATORS_PROPERTIES_FIELDS)``` usage and just replace it with ```REPLICATORS_PROPERTIES_FIELDS```.